lib/constants.js

```diff
@@ -6,3 +6,7 @@ module.exports.API_GEOCODER_REVERSE = compile('${endpoint}/v4/geocode/${dataset}
 module.exports.API_DIRECTIONS = compile('${endpoint}/v4/directions/${profile}/${encodedWaypoints}.json?${query}');
 module.exports.API_SURFACE = compile('${endpoint}/v4/surface/${mapid}.json?${query}');
 module.exports.API_MATCHING = compile('${endpoint}/matching/v4/${profile}.json?${query}');
+module.exports.API_DATASET_DATASETS = compile('${endpoint}/datasets/v1/${user}?${query}');
```

Comment:
We have been trying to avoid `user` in this context because it is ambiguous with the user that's authenticating the request. Could we use `{owner}` instead?